Lib/test/test_email/test_email.py

```diff
@@ -19,24 +19,25 @@
 import email.policy
 
 from email.charset import Charset
-from email.header import Header, decode_header, make_header
-from email.parser import Parser, HeaderParser
 from email.generator import Generator, DecodedGenerator, BytesGenerator
+from email.header import Header, decode_header, make_header
+from email.headerregistry import HeaderRegistry
 from email.message import Message
 from email.mime.application import MIMEApplication
 from email.mime.audio import MIMEAudio
-from email.mime.text import MIMEText
-from email.mime.image import MIMEImage
 from email.mime.base import MIMEBase
+from email.mime.image import MIMEImage
 from email.mime.message import MIMEMessage
 from email.mime.multipart import MIMEMultipart
 from email.mime.nonmultipart import MIMENonMultipart
-from email import utils
-from email import errors
+from email.mime.text import MIMEText
```

Comment:
Please move these back up to the `from email.foo` section.

Reply:
@warsaw I think you are going to have to spell it out for me because I did not move any imports away from any of the pre-established sections. Do you want me to just make the imports exactly how they were before but add `from email.headerregistry import HeaderRegistry` just after `from email.header...` but just before `from email.parser`?

Reply:
Yeah, unfortunately it's not obvious and much of this was inherited for when this library was maintained separately.  Basically, it's the `from email.foo import` imports come first, followed by the `from email import` imports.  Sort each individual blocks alphabetically.

If that doesn't make sense and you want to give me write access to the branch, I'll be happy to clean it up to help get this PR landed.

Reply:
@warsaw Perhaps you could copy and paste what you want here?

Reply:
@oda-gitso I find it difficult to grok the diff in GH's interface, so I just used `gh` CLI to check out the PR branch.  Everything looks great.  Thanks!

Reply:
@warsaw Thanks a lot for your help/time!